apollo-gradle-plugin/src/main/groovy/com/apollostack/android/gradle/ApolloExtension.groovy

```diff
@@ -0,0 +1,34 @@
+package com.apollostack.android.gradle
+
+import com.google.common.collect.ArrayListMultimap
+import com.google.common.collect.Multimap
+import org.gradle.api.Project
+import org.gradle.api.tasks.util.PatternSet
+
+class ApolloExtension {
+  private final Project project
+  static String NAME = "apollo"
+  String graphqlPath
+  // Maps GraphQL query files' relative paths to a sourceSet directory
+  private Multimap<String, String> cachedFiles
+
+  ApolloExtension(Project project, String sourceSet) {
+    this.project = project
+    this.graphqlPath = "src/$sourceSet/graphql"
```

Comment:
For example, `src/main/graphql`, `src/debug/graphql`, etc

This sets the default but we can allow the user to have a custom one through extensions in the Gradle DSL.
This should be valid for all cases since the compiler enforces the `src/x/graphql` structure anyways.